PatientRegistrationApp/src/components/HouseholdFollowUp.vue

```diff
@@ -10,19 +11,17 @@
 		<v-row justify="center">	
 			<v-col cols="12">
 				<v-card class="d-flex flex-wrap" flat>
-					<v-card class="pa-2" flat min-width=33%>
-						<div class="font-weight-medium primary--text">Household Member #1</div>
-						<div> QR Code Placeholder</div>
-						<div class="font-weight-medium">Name:  <span class="font-weight-regular">{{dataHouseholdPersonalInfo[0].familyName}}, 
-						{{dataHouseholdPersonalInfo[0].givenName}} {{dataHouseholdPersonalInfo[0].middleName}} {{dataHouseholdPersonalInfo[0].suffix}}</span></div>
-					</v-card>
-				<template v-for="index in getNumberOfHouseholdMembers()-1">
+				<template v-for="index in getNumberOfHouseholdMembers()">
 						<v-card class="pa-2" flat min-width=33%
 						:key="index">
-							<div class="font-weight-medium primary--text">Household Member #{{index+1}}</div>
-							<div> QR code Placeholder</div>
-							<div class="font-weight-medium">Name:  <span class="font-weight-regular">{{dataHouseholdPersonalInfo[index].familyName}}, 
-							{{dataHouseholdPersonalInfo[index].givenName}} {{dataHouseholdPersonalInfo[index].middleName}} {{dataHouseholdPersonalInfo[index].suffix}}</span></div>
+							<div class="font-weight-medium primary--text">Household Member #{{index}}</div>
+							<v-row><div>
+							<vue-qrcode
+								v-bind:value="toQRValue(dataHouseholdPersonalInfo[index-1])"
+								v-bind:errorCorrectionLevel="correctionLevel" />
```

Comment:
What's the correction level?

Reply:
The error correction level adds a certain amount of “backup” data depending on how much damage the QR code is expected to suffer in its intended environment. I used H which is the highest (up to 30% damage). However if you think it's not that necessary I believe the QR code generator can function without it

Reply:
No that's a good idea. Let's stick with it!